src/main/java/sqltoregex/deparser/OrderByDeParserForRegEx.java

```diff
@@ -124,15 +107,15 @@ public ExpressionVisitor getExpressionVisitor() {
      * @param expressionVisitor {@link ExpressionVisitor}
      */
     public void setExpressionVisitor(ExpressionVisitor expressionVisitor) {
-        this.expressionDeParserForRegEx = expressionVisitor;
+        this.expressionDeParserForRegEx = (ExpressionDeParserForRegEx) expressionVisitor;
```

Comment:
since this is no Override, wouldn't it be possible to simply request an ExpressionDeParserForRegEx as method parameter?

Reply:
done